README.md

```diff
@@ -10,6 +10,18 @@ Automation has been introduced for creating new config files. New deployers can
 Info for parthers: [in the docs](https://github.com/e-mission/e-mission-docs/tree/master/docs/use/start_a_project.md)
 Info for developers: [in the docs](https://github.com/e-mission/e-mission-docs/tree/master/docs/dev/future/more_custom_auto_config.md)
 
+Config file submission will begin the process of creating an admin dashboard and associated user pool for the project. In order to log in, users will need to generate an account. This process will be automated, but may occasionally need to be run manually if there are errors in the initial config submission. To generate an account:
+
+#Run the following command in terminal if boto3 is not installed
+
+`pip install boto3`  
+
+#Run the email-config.py script, and pass the name of the file in as an argument:
+
+`python email-config.py configfile.nrel-op.json`
```

Comment:
**future cleanup**: Support a path to the filename. That is more unix-like and allows us to test out temporary files in non-standard locations as well. 
(the `os.path` module should help you parse out the filename from the path).

Reply:
I am not sure what you mean by this. Support a path as an argument for running in command line?

Reply:
Right now, we specify the filename - e.g. ``python email-config.py wyoming.nrel-op.json` and then you assume that the file location is `../config/wyoming.nrel-op.json`. 
https://github.com/e-mission/nrel-openpath-deploy-configs/pull/45/files#diff-406c93e4576589a3f4a4084699b1f01bf7c19a7b5cf7a3905561c2634431a499R18

I'm asking you to allow the user to specify the full filename (so I can have a copy of wyoming as `/tmp/wyoming-test-me-me-me.json` and use it instead.

The GH action would then invoke the script with `../config/<filename>`.